### DISCUSSION TITLE
UNO Project Template+.Net7+IOS/Android - Build actions missing

### DISCUSSION THREAD

**Discussion by MCAleckson** [unoplatform/uno#11299] [Q&A]
I have a few UNO apps in production with a few hundred thousand users (IOS and Android).  I built these apps over the last 2.5 years using the original project templates.

It's time to move the code base up to .Net 7, and I'd like to update the project file structure.

So I updated everything in my VS environments, and commenced to create a new Uno project.  Tried several times actually.  I'm using the  "Uno Platform App" that creates all the mobile projects in one head.

I use several NuGet packages for Google and Firebase services, AWS backend services and many more.  

Copied all the code from my working old project into the new.  All the manifest entries, blah, blah.  Updated NuGets to run .Net7 code.

Runs on Android.  IOS bombs spectacularly.  So I'm digging as usual now.

A problem is that the build actions I'm used to using for files like GoogleServicesJSON, GoogleService-Info, resource files, other plists - like for IOS "BundleResource" - don't appear.

So the project structure is suspect to me.

I feel like I've got to be missing something fundamental here.  And the [linked detail documentation](https://platform.uno/docs/articles/get-started-winui3.md) for dotnet new is broken and unreadable.

Any pointers you can give are much appreciated.

Thank you very much.

**Comment by jeromelaban** [comment]
Thanks for reaching out. The new project structure uses a separate project for building applications (Compared to a **Shared Project** found in previous templates), and files placed in that project may not end up being packaged as `BundleResource` (this is not supported by .NET altogether). 

That being said, files set as `Content` build action will be available in the final package, and generally be used through [`StorageFile.GetFileFromApplicationUriAsync`](https://platform.uno/docs/articles/features/file-management.html?q=storage#support-for-storagefilegetfilefromapplicationuriasync).

**Comment by sasakrsmanovic** [comment]
Thanks for reaching out! 
I'll let the team troubleshoot the project structure with you. 

As for the doc link you shared, we are on it now and will get it fixed ASAP.

Separately from all that, I would LOVE to hear from you to see your app. As open-source maintainers we don't know who is using us and for what. It always gives us great energy boost to hear from our users. If you don't mind please reach out to sasha@platform.uno

**Reply by jeromelaban** [reply]
@MCAleckson About the link to the missing documentation, could you tell use where you found this particular link?

**Reply by MCAleckson** [reply]
Jerome, the link is in the Uno documentation -> Get Started -> Development Environment -> dotnet new -> "Find Detailed Instructions Here".

And thank you Sasha - we've actually already spoken at some length about my apps, they're used in the child/athlete protection space.  I appreciate your interest!
